modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -9,6 +9,23 @@ The default installation of metering configures Hive to use an embedded Java dat
 
 There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
 
+
+You will create your MySQL or Postgres instance with a username and secret. Then either in the OpenShift CLI or by making a yaml file, you will make a secret. The `secretName` you use for this secret needs to map to your Metering CR `spec.hive.spec.config.db.secretName`.
```

Comment:
A couple of things here:
- A bit confused as to what you are trying to convey with the first sentence. Did you mean `... with a username and password.`?
- We should use `MeteringConfig` instead of `Metering` and spell out custom resource.
- If the goal with this block is to expose (or provide an overview) for using the `secretName` configuration, as opposed to the username + password one, we should just describe the need for creating a secret containing the mysql or postgresql's database credentials. We should make the requirement for the username and password data keys to base64-encrypted explicit.